dolly-backend-app/src/main/java/no/nav/dolly/bestilling/dokarkiv/domain/DokarkivRequest.java

```diff
@@ -1,23 +1,25 @@
 package no.nav.dolly.bestilling.dokarkiv.domain;
 
-import static java.util.Objects.isNull;
-
-import java.util.ArrayList;
-import java.util.List;
-
 import com.fasterxml.jackson.annotation.JsonInclude;
-
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
+import lombok.ToString;
 import no.nav.dolly.domain.resultset.dokarkiv.RsDokarkiv;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.isNull;
+
 @Getter
 @Setter
 @NoArgsConstructor
 @AllArgsConstructor
+@ToString
```

Comment:
Du kan evt benytte lombok Data som erstatning for Getter Setter og ToString

Reply:
Sant, godt sett 😄